app/controllers/ops_controller.rb

```diff
@@ -20,6 +20,10 @@ def self.model
     Tenant
   end
 
+  def show
+    raise NotImplementedError,  _("Action show is not implemented.")
```

Comment:
Why?

Reply:
yes - there is used `GenericShowMixin` but not for `show` method.`GenericShowMixin ` here is included to use `nested_list` (https://github.com/ManageIQ/manageiq-ui-classic/blob/2712c26e5a6618ce3375eabf4cfab95dad000467/app/controllers/ops_controller.rb#L858)

so I wanted to say that method `show` is not implemented in this controller despite we have in `GenericShowMixin` here. Not sure - or maybe leave here  some comment to describe it ? 

it was kind of confusing for me to find reason and I needed to ask author  - there is `show` route and there is `show` method but it is here basically by coincidence - we don't use `show` method on opscontroller.

Reply:
I don't understand this, if you have a dead route, you should not add a method that throws an error to fix the dead route. The solution is to get rid of the dead route.

Reply:
ok I removed it.

Reply:
or I can leave show route here and add method `show` which will just redirect it to index.

Reply:
I put back `show` method method `show` is used.

so this can be ignored 

thanks